### PR TITLE
Add contributing page for designers

### PR DESCRIPTION
**Pull request by xander-marjoram**
After some discussions with Jordi we're also looking to update the breakpoint at which the nav disappears - from mid to wide. 

This gives the page content a bit more breathing room, especially when we have the 240px image and text beside it. To be consistent we'll update it across the entire site.

_Note: this isn't really picked up by the visual regression tests because we test wider-than-wide (1280px)._

### CODE REVIEW

**Comment by jamieomaguire** [comment]
Not a pressing concern but I do think we should revisit this approach for the mid-way breakpoints. The cards are pretty large and I think look a bit off being so stretched. You could even keep them at a max-width of, say, 280px and start displaying them in rows or something. It just isn't a great use of space as is in my opinion

File: apps/pie-microsite/src/_11ty/shortcodes/mediaElementList.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    const listItems = JSON.parse(data).items;
+
+    return `<div class="c-mediaElementList">
+    ${listItems.map(({
+        headingLevel = 3, headingText, imgSrc, imgSrcNarrow, text
+    }) => {
+        const hx = `h${headingLevel}`;
+        return `<div class="c-mediaElement">
+            <picture>
+                <source srcset="${imgSrc}" media="(min-width: 600px)">
```
Comment (jamieomaguire) [inline apps/pie-microsite/src/_11ty/shortcodes/mediaElementList.js:11]: nitpick: can we pop `600px` in a variable? if we do this elsewhere then I'd prefer this approach everywhere

File: .changeset/kind-ways-teach.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"pie-microsite": minor
+---
+
+Update sidebar breakpoint to wide instead of mid
```
[the remark this answers is not in the document]
Comment (xander-marjoram) [reply .changeset/kind-ways-teach.md:5]: I spoke to @siggerzz and he said it was okay, they should all get combined into one

[the remark this answers is not in the document]
Comment (jamieomaguire) [reply .changeset/kind-ways-teach.md:5]: yeah I'm completely wrong haha - sorry!

**Review by fernandofranca — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Pull request merged**